src/main/java/jupyter/Registration.java

```diff
@@ -36,6 +36,22 @@
   private final Map<Class<?>, Displayer<?>> displayers = new HashMap<>();
   private Displayer<Object> defaultDisplayer = ToStringDisplayer.get();
   private String[] mimeTypes = null;
+  private final boolean registerAsDisplayData;
+
+  private void init() {
+    if (registerAsDisplayData) {
+      add(AsDisplayData.class, AsDisplayData.displayer);
+    }
+  }
+
+  public Registration() {
+      this(true);
```

Comment:
In what case would it be reasonable to not register the displayer for `AsDisplayData`?

Reply:
None really. Removed that possibility.